### DISCUSSION TITLE
Pipx isn't mentioned as installation alternative anymore

### DISCUSSION THREAD

**Discussion by mbeko** [pypa/pipenv#6204] [General]
I noticed that Pipx isn't mentioned anymore in the [documentation as a way to install Pipenv](https://pipenv.pypa.io/en/latest/installation.html). I found out that it has been removed with the [migration from `install.rst` to `installation.md`](https://github.com/pypa/pipenv/commit/761a03d088b9ac7d568acb849c8bd8131f473593#diff-379774f5c6e537f2001c951863c6ac577b56eb5c6e61cc581dce95143760ff36) but I didn't see any explanation.

I wonder what the reason for the removal was. I find Pipx very useful to install globally used CLI applications like Pipenv. Maybe @matteius can tell more as the author of the commit?

**Comment by matteius** [comment]
@mbeko At the time we had a number of issue reports around installation, and I think it was around the time that half or more of the imports for pip came from the user installed pip, so we were trying to reduce variables in the issues being reported.   I think if you check the readme it still mentions pipx installation, or it does somewhere in the docs as I encountered it recently.   pipx is supported, not something I regularly use which is probably why I had reduced it at that time.   We are looking for help to improve the docs, so feel free to open a PR or two if that is something you are willing to do.
